Content.Server/Salvage/SalvageMagnetComponent.cs

```diff
@@ -47,6 +47,31 @@ public sealed class SalvageMagnetComponent : SharedSalvageMagnetComponent
         [DataField("magnetState")]
         public MagnetState MagnetState = MagnetState.Inactive;
 
+        /// <summary>
+        ///     How long it takes for the magnet to pull in the debris (in seconds)
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("attachingTime")]
+        public int AttachingTime;
```

Comment:
There should be a blank line between all the class fields.

Reply:
Sorry, fixed